.idea/codeStyles/Project.xml

```diff
@@ -151,10 +151,9 @@
       <option name="KEEP_BLANK_LINES_IN_CODE" value="1" />
       <option name="KEEP_BLANK_LINES_BEFORE_RBRACE" value="1" />
       <indentOptions>
-        <option name="INDENT_SIZE" value="2" />
         <option name="CONTINUATION_INDENT_SIZE" value="4" />
         <option name="TAB_SIZE" value="2" />
       </indentOptions>
     </codeStyleSettings>
   </code_scheme>
-</component>
+</component>
```

Comment:
Hi @vinitamurthi -- should add newline at EOF, here and elsewhere? I think this is something the linter should catch, btw.

Reply:
The linter currently works only on kotlin files, I will add separate linters to handle other file formats

Reply:
Actually I don't think we should check in changes to these config files, this file and the below file was created because I made changes to my android studio config for kotlin (I changed the indent to 4 spaces in my android studio). @BenHenning what do you think? Should we make changes to the android studio preferences?

Reply:
Ah ok, thanks for explaining. I have no idea, sorry, deferring to @BenHenning. (But, if we keep the file, let's make sure there's a newline at EOF for consistency.)

Reply:
Talked to @BenHenning  about this, we are thinking of allowing 2 space indents instead of 4. So I will close this PR and create a new one with the linting changes

Reply:
@rt4914 FYI